backend/service/k8s/cache.go

```diff
@@ -118,45 +118,46 @@ func (s *svc) cacheFullRelist(ctx context.Context, cluster string, lwPods, lwDep
 		case <-ticker.C:
 			pods, err := lwPods.List(metav1.ListOptions{})
 			if err != nil {
-				s.log.Error("Unable to list pods to populate Kubernetes cache", zap.Error(err))
-			}
-
-			podItems := pods.(*corev1.PodList).Items
-			for i := range podItems {
-				pod := &podItems[i]
-				if err := ApplyClusterMetadata(cluster, pod); err != nil {
-					s.log.Error("Unable to apply cluster name to pod object", zap.Error(err))
+				s.log.Error("Unable to list pods to populate Kubernetes cache", zap.String("cluster", cluster), zap.Error(err))
+			} else {
```

Comment:
i think you can call `break` instead of nesting everything

Reply:
One resource might fail, but the next one could succeeded. The API servers latencies are variable by resource.

Reply:
ah, i see now :+1: